src/Application/Model/Ticket.php

```diff
@@ -658,6 +658,10 @@ public static function countByNextState($project, $ticketType, $ticketState) {
 					'ticket_state_next = ?',
 					[$ticketState]
 				)
+				->where(
+					'failed IN  (?, ?)',
+					[false, $includeFailed]
+				)
```

Comment:
I see why you went there, but I think we're better off, when we conditionally call `where`.

Reply:
In general I'm not in favor of hiding implicit conditions within generic looking count-functions. At least not without a comment explaining why the condition makes sense.
Generic model functions need to be self-explanatory, either just by its name and its parameters or with additional comments, but I don't want to be required to lookup it's uses to figure out why it behaves like it does.
So either a comment, or move the condition to the function call (via adding a new parameter)?!